js/main.js

```diff
@@ -1,33 +1,141 @@
-//Функция, возвращающая случайное целое число из переданного диапазона включительно.
-const getRandom = (min, max) => {
-  if (max === undefined || min === undefined) {
-    return 'Введите все аргументы функции';
-  }
-  if (max === min) {
-    return 'Диапазоном не может быть одно число';
-  }
-  if (min > max) {
-    return 'Введите корректный диапазон "от...до"';
+const SIMILAR_ADVERTISEMENTS_COUNT = 10;
+const MIN_PRICE = 5000;
+const MAX_PRICE = 100000;
+const MAX_ROOM_COUNT = 100;
+const MAX_GUESTS_COUNT = 10;
+const LATITUDE_FROM = 35.65000;
+const LATITUDE_TO = 35.70000;
+const LONGITUDE_FROM = 139.70000;
+const LONGITUDE_TO = 139.80000;
+const TWO_DIGIT_NUMBER = 10;
+const DIGITS_AFTER_POINT = 5;
+
+const HEADLINE = [
+  'Квартирка для котов лентяев',
+  'Дворец для состоятельных мурчал',
+  'Сдаю гараж с мышами',
+  'Дом для многодетных котов',
+  'Валерианин дом',
+];
+
+const DESCRIPTION_OF_HOUSING = [
+  'Ночлежка для ночных котобродяг',
+  'Бесплатный вискас, молоко, когтеточка, массаж и мурмяу',
+  'Отель без хозяев, без котодетей, без котообязательств',
+  'Место для котов без хвостов и совести',
+  'Место чтоб залечь на дно, если натворил кое-каких дел',
+  'Аппартаменты для котобоссов и котобоссок',
+];
+
+const PHOTO_OF_HOUSING = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
+];
+
+const TYPE_HOUSING = [
+  'palace',
+  'flat',
+  'house',
+  'bungalow',
+  'hotel',
+];
+
+const FEATURES_OF_HOUSING = [
+  'wifi',
+  'dishwasher',
+  'parking',
+  'washer',
+  'elevator',
+  'conditioner'
+];
+
+const CHECK_IN_TIME = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+
+const CHECK_OUT_TIME = CHECK_IN_TIME.slice();
+
+const createArrayIdAvatar = () => {
```

Comment:
```suggestion
const createIndexes = () => {
```

Во-первых, лучше не использовать служебные слова в названиях функций и переменных; во-вторых, эта функция ничего не знает про аватары, потому что внутри неё ничего нет про аватары. Также, она ничего не знает про id. Всё, что твоя функция делает, это создаёт массив чисел от 0 до 10